ui/pages/snaps/snap-view/snap-home.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 
 import { SnapHomeRenderer } from '../../../components/app/snaps/snap-home-page/snap-home-renderer';
 import { Box } from '../../../components/component-library';
-import { BlockSize } from '../../../helpers/constants/design-system';
+import { BackgroundColor } from '../../../helpers/constants/design-system';
 
 function SnapHome({ snapId }) {
   return (
-    <Box height={BlockSize.Full}>
+    <Box backgroundColor={BackgroundColor.backgroundDefault}>
```

Comment:
I think we would want this to stay full height
<img width="389" alt="Screenshot 2024-08-28 at 12 01 58" src="https://github.com/user-attachments/assets/02e06601-1340-43f6-a982-ebf77fd212d8">

Reply:
Oh, why is it not... 🤔 
I had problem with this, because background was cut in half when content is longer. So, removing the height and setting it to auto fixed that. But, I was not testing with less content. Will need to find another solution for this.

Reply:
Fixed both scenarios and ensured that width and height are going to stay at 100% in the parent boxes. Adding `overflow: auto` to the container Box resolves the issue with background being cut in half when home page content is longer than the initial size of the popup window.